source/codegen/metadata/nixnetsocket/custom_proto.mako

```diff
@@ -54,11 +54,23 @@ message Linger {
   int32 l_linger = 2;
 }
 
+message IPMReq {
+  uint32 imr_multiaddr = 1;
```

Comment:
await: I introduced message types for nxin_addr and nxin6_addr (IPv4Addr and IPv6Addr; though I may change the names later) in #616 , I like that both (a) for consistency with the C API, (b) to provide meaning to disorienting data, and (c) to make it easier to copy to/from request/response objects. I'll admit it's not the killerest feature though. Can you check out that PR. And then update to use those and/or make a counter-argument?

Reply:
Yeah I will want to update to have these messages use those new addr types you introduced for consistency with the API that makes sense.

Reply:
I've updated everything to use the new IPv4 and IPv6 Addrs